reconcile/change_owners/change_types.py

```diff
@@ -351,3 +352,10 @@ class ChangeTypeContext:
     change_type_processor: ChangeTypeProcessor
     context: str
     approvers: list[Approver]
+
+
+JSON_PATH_ROOT = "$"
+
+
+def change_path_covered_by_allowed_path(changed_path: str, allowed_path: str) -> bool:
+    return changed_path.startswith(allowed_path) or allowed_path == JSON_PATH_ROOT
```

Comment:
I'm probably missing some context here but wouldn't this conditional always return true since every JSONPath object starts with `$`? Is this what you're intending for?

Reply:
that's what i thought as well, that is the reason this situation was handled correctly until now. it seems that is not really mandatory in JSONPath, at least the python library seems to hand out paths without the "$" prefix

Reply:
Oh, interesting. Did not know that!